src/material-experimental/mdc-list/list-base.ts

```diff
@@ -53,22 +48,26 @@ export abstract class MatListItemBase implements AfterContentInit, OnDestroy, Ri
 
   private _rippleRenderer: RippleRenderer;
 
-  constructor(protected _element: ElementRef, protected _ngZone: NgZone, listBase: MatListBase,
-              platform: Platform) {
-    const el = this._element.nativeElement;
+  protected constructor(public _elementRef: ElementRef<HTMLElement>, protected _ngZone: NgZone,
+                        listBase: MatListBase, platform: Platform) {
     this.rippleDisabled = listBase._isNonInteractive;
     if (!listBase._isNonInteractive) {
-      el.classList.add('mat-mdc-list-item-interactive');
+      this._elementRef.nativeElement.classList.add('mat-mdc-list-item-interactive');
     }
     this._rippleRenderer =
-        new RippleRenderer(this, this._ngZone, el, platform);
-    this._rippleRenderer.setupTriggerEvents(el);
+        new RippleRenderer(this, this._ngZone, this._elementRef.nativeElement, platform);
+    this._rippleRenderer.setupTriggerEvents(this._elementRef.nativeElement);
   }
 
   ngAfterContentInit() {
     this._monitorLines();
   }
 
+  ngOnDestroy() {
+    this._subscriptions.unsubscribe();
```

Comment:
Optional: I missed this before, but we've been moving towards using a `_destroyed` subject with `takeUntil(this._destroyed)` instead of keeping track of subscriptions

Reply:
We have? I think that's more susceptible to memory leaks because you have to think about the order you apply it compared to other operators, e.g. if for some reason we did:

```
.pipe(takeUntil(this._destroyed), delay(100)).subscribe(...)
```

We could get an event, then 1ms later the component is destroyed, then 99ms after that we try to do something in response to it. I'd prefer to use the more fool-proof `subscriptions.unsubscribe()` style if we're going to converge on one.

This isn't just theoretical, I remember it coming up for at least one of our components at some point

Reply:
Maybe a topic for the team meeting, but IMO the `takeUntil` approach is easier to read (and is maybe more "idiomatic" rxjs, but I don't know that that's important in and of itself)

Reply:
I'll leave this as is for now, we can change it if needed after the team meeting